baremetal/metal3machine_manager.go

```diff
@@ -783,7 +788,12 @@ func (m *MachineManager) chooseHost(ctx context.Context) (*bmov1alpha1.BareMetal
 	availableHostsWithNodeReuse := []*bmov1alpha1.BareMetalHost{}
 
 	for i, host := range hosts.Items {
-		if host.Spec.ConsumerRef != nil && consumerRefMatches(host.Spec.ConsumerRef, m.Metal3Machine) {
+		ok, err1 := consumerRefMatches(host.Spec.ConsumerRef, m.Metal3Machine)
```

Comment:
is some linter forcing not to use `err` here?
 lets use `err` is possible.

Reply:
yes. I removed checking parsing errors.